index.js

```diff
@@ -28,11 +29,15 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater(num) {
+  console.log("Man I sure am thirsty")
+  a = --num 
+  console.log("Ahh that hits the spot")
+  return a
+}
```

Comment:
Nice work on this! Just be careful on line 34 here, if you declare a variable without `let` or `const` it will become a global variable (even if you declare it inside a function).